Makefile

```diff
@@ -125,7 +128,8 @@ e2e-test-all: e2e-test-ibc-success-evm \
 	e2e-test-eibc-corrupted-memo-evm \
 	e2e-test-eibc-excessive-fee-evm \
 	e2e-test-eibc-fulfillment-evm \
-    e2e-test-eibc-fulfill-no-balance-evm \
+  e2e-test-eibc-fulfillment-evm-2-RAs \
+  e2e-test-eibc-fulfill-no-balance-evm \
```

Comment:
Same cmt above, if we are lacking of the adding this name of test, we should separate it to another pr to ez tracking @hoangdv2429

Reply:
same as above